### PR TITLE
Commerce 10088

### PR DESCRIPTION
**Pull request by brianikim**
From @victorcdp,

> Related Issue: [COMMERCE-10088](https://issues.liferay.com/browse/COMMERCE-10088)
> 
> Simple task to remove the Configuration from Product as it's been deprecated by ProductConfiguration

@andrea-ale-sbarra

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:commerce

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java [hunk 1]
```diff
@@ -784,7 +784,8 @@ private CPDefinition _updateNestedResources(
 
 		// Product configuration
 
-		ProductConfiguration productConfiguration = product.getConfiguration();
+		ProductConfiguration productConfiguration =
+			product.getProductConfiguration();
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java:789]: you should also remove the underneath if

Comment (andrea-ale-sbarra) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java:789]: most probably

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java (hunk 1 above)
Comment (victorcdp) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java:789]: Hi @andrea-ale-sbarra, could you explain to me why should this be removed? I took a look and every other nested resource has a similar if. As i understand this is just to update the CpDefinitionInventory with the new product configuration.

Comment (andrea-ale-sbarra) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java:789]: Ehi @victorcdp forget about that I was mislead by  the getter methd that you changed I thought the underneath if was something old to change too, but it was not.

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
Hi @brianikim could you please fix the conflicts in this pr? Thanks in advance!

**Comment by brianikim** [comment]
@victorcdp , fix the issues in this PR and resend. Thanks.

**Pull request closed without merging**